mslib/mscolab/server.py

```diff
@@ -556,14 +556,13 @@ def error413(error):
 @verify_user
 def create_operation():
     path = request.form['path']
-    content = request.form.get('content', None)
+    content = request.form.get('content')
```

Comment:
, None

the filemanager reacts on None with False. 

We should keep this here